diff -dur -x '*~' mcabber-0.9.3.orig/configure.ac mcabber-0.9.3/configure.ac
--- mcabber-0.9.3.orig/configure.ac	2007-06-19 22:09:57.000000000 +0200
+++ mcabber-0.9.3/configure.ac	2008-04-07 10:02:11.000000000 +0200
@@ -174,7 +174,7 @@
 
 # Export $datadir to the source tree.
 if test x"${datadir}" != x""; then
-    AC_DEFINE_DIR(DATA_DIR, "${datadir}", [Data files directory])
+    AC_DEFINE_DIR(DATA_DIR, [datadir], [Data files directory])
 fi
 
 AC_ARG_ENABLE(debug,
